### PR TITLE
[SYCLomatic] Make some query-api-mapping cases more readable.

### PR DESCRIPTION
**Pull request by tangjj11**
Signed-off-by: Tang, Jiajun jiajun.tang@intel.com

### CODE REVIEW

**Review by zhiweij1 — APPROVED** [review]

File: clang/examples/DPCT/Runtime/cudaGetChannelDesc.cu [hunk 1]
```diff
@@ -1,6 +1,7 @@
 void test(cudaChannelFormatDesc *pc) {
+  // TODO: a's type need to be changed to cudaArray_const_t
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Runtime/cudaGetChannelDesc.cu:2]: Please fix the todo

File: clang/examples/DPCT/Driver/cuMemcpy2D.cu [hunk 2]
```diff
@@ -1,5 +0,0 @@
-void test(const CUDA_MEMCPY2D *pm) {
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Driver/cuMemcpy2D.cu:1]: while remove 5 sample files. (4 in Driver folder and 1 in Runtime folder)

File: clang/examples/DPCT/Driver/cuMemcpyDtoH.cu [hunk 3]
```diff
@@ -1,5 +1,6 @@
-void test(void *pv, CUdeviceptr pd, size_t s) {
+void test(void *pv, size_t s) {
   // Start
-  cuMemcpyDtoH(pv /*void **/, pd /*CUdeviceptr*/, s /*size_t*/);
+  CUdeviceptr pd;
+  cuMemcpyDtoH(pv /*void **/, pd, s /*size_t*/);
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Driver/cuMemcpyDtoH.cu]: pv ==> dstHost, 
pd==>  srcDevice

File: clang/examples/DPCT/Runtime/cudaMalloc3D.cu [hunk 4]
```diff
@@ -1,5 +1,6 @@
-void test(cudaPitchedPtr *pp, cudaExtent e) {
+void test(cudaPitchedPtr *pp) {
   // Start
-  cudaMalloc3D(pp /*cudaPitchedPtr **/, e /*cudaExtent*/);
+  cudaExtent e;
+  cudaMalloc3D(pp /*cudaPitchedPtr **/, e);
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Runtime/cudaMalloc3D.cu]: pp is a strange name

File: clang/examples/DPCT/Runtime/cudaGetChannelDesc.cu (hunk 1 above)
Comment (tangjj11) [reply clang/examples/DPCT/Runtime/cudaGetChannelDesc.cu:2]: Now the cudaArray_const_t cannot be migrated correctly.

File: clang/examples/DPCT/Driver/cuMemcpy2D.cu (hunk 2 above)
Comment (tangjj11) [reply clang/examples/DPCT/Driver/cuMemcpy2D.cu:1]: Now the CUDA_MEMCPY2D cannot be migrated correctly.

File: clang/examples/DPCT/Runtime/cudaMemcpy2DArrayToArray.cu [hunk 5]
```diff
@@ -1,9 +1,11 @@
-void test(cudaArray_t a, size_t s1, size_t s2, cudaArray_const_t ac, size_t s3,
-          size_t s4, size_t s5, size_t s6, cudaMemcpyKind m) {
+void test(size_t s1, size_t s2, size_t s3, size_t s4, size_t s5, size_t s6) {
+  // TODO: src's type need to be changed to cudaArray_const_t
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Runtime/cudaMemcpy2DArrayToArray.cu:2]: fix the todo

File: clang/examples/DPCT/Runtime/cudaMemcpy2DFromArray.cu [hunk 6]
```diff
@@ -1,8 +1,9 @@
-void test(void *pv, size_t s1, cudaArray_const_t a, size_t s2, size_t s3,
-          size_t s4, size_t s5, cudaMemcpyKind m) {
+void test(void *dst, size_t s1, size_t s2, size_t s3, size_t s4, size_t s5) {
+  // TODO: src's type need to be changed to cudaArray_const_t
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Runtime/cudaMemcpy2DFromArray.cu:2]: fix the todo

File: clang/examples/DPCT/Driver/cuMemcpyDtoH.cu (hunk 3 above)
Comment (tangjj11) [reply clang/examples/DPCT/Driver/cuMemcpyDtoH.cu]: changed to pHost and pDev.

File: clang/examples/DPCT/Runtime/cudaMalloc3D.cu (hunk 4 above)
Comment (tangjj11) [reply clang/examples/DPCT/Runtime/cudaMalloc3D.cu]: changed to `pitch`

File: clang/examples/DPCT/Runtime/cudaMemcpy2DArrayToArray.cu (hunk 5 above)
Comment (tangjj11) [reply clang/examples/DPCT/Runtime/cudaMemcpy2DArrayToArray.cu:2]: Now the cudaArray_const_t cannot be migrated correctly.

File: clang/examples/DPCT/Runtime/cudaMemcpy2DFromArray.cu (hunk 6 above)
Comment (tangjj11) [reply clang/examples/DPCT/Runtime/cudaMemcpy2DFromArray.cu:2]: Now the cudaArray_const_t cannot be migrated correctly.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**